notebooks/visualization/matplotlib-intermediate.ipynb

```diff
@@ -0,0 +1,344 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## 5. Plot with Contour\n",
+    "\n",
+    "**contour** creates contours around the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create one figure for two plots\n",
+    "fig = plt.figure(figsize=(15,6))\n",
+    "\n",
+    "# Create a simple contour\n",
+    "ax = fig.add_subplot(1, 2, 1)\n",
+    "ax.contour(X, Y, Z)\n",
+    "\n",
+    "# Create a second contour with labels\n",
+    "ax2 = fig.add_subplot(1, 2, 2, sharex=ax, sharey=ax)\n",
+    "c = ax2.contour(X, Y, Z, levels=np.arange(-2, 2, 0.25))\n",
+    "ax2.clabel(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## 6. Plot with Contourf\n",
+    "\n",
+    "**contourf** creates filled contours around data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "c = ax.contourf(X, Y, Z)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## 7. Combine Imshow and Contour\n",
+    "\n",
+    "Create a figure using imshow and contour that is a heatmap in the colormap of your choice. Overlay black contours with a 0.5 contour interval."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Cell content replaced by load magic replacement.\n",
+    "fig, ax = plt.subplots()\n",
+    "im = ax.imshow(Z, interpolation='bilinear', cmap='PiYG', origin='lower', extent=[-3, 3, -3, 3])\n",
+    "c = ax.contour(X, Y, Z, levels=np.arange(-2, 2, 0.5), colors='black')\n",
+    "ax.clabel(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---\n",
+    "\n",
+    "## See also\n",
+    "\n",
+    "Documentation for:\n",
+    "\n",
+    "* [matplotlib.pyplot](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.html)\n",
+    "* [matplotlib.pyplot.figure](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.figure.html)\n",
+    "* [matplotlib.pyplot.axes](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.axes.html)\n",
+    "* [matplotlib.pyplot.imshow](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.imshow.html)\n",
+    "* [matplotlib.pyplot.contour](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.contour.html)\n",
+    "* [matplotlib.pyplot.contourf](https://matplotlib.org/3.3.3/api/_as_gen/matplotlib.pyplot.contourf.html)\n",
+    "\n",
+    "\n",
+    "### Related Notebooks\n",
+    "\n",
+    "* Matlibplot: Basics\n",
```

Comment:
Here too

Reply:
Sorry, I should have done that sooner myself.  I went through and searched for 'matlibplot' in all of my notebooks and found a few more misspellings as well, that are fixed in the latest PR.

Reply:
No worries!